itou/employee_record/management/commands/sanitize_employee_records.py

```diff
@@ -80,7 +76,9 @@ def _check_missed_notifications(self, dry_run):
             )
         )
 
-        self.stdout.write(f" - found {len(employee_record_with_missing_notification)} missing notification(s)")
+        self.logger.info(
+            "found %d missing employee records notifications", len(employee_record_with_missing_notification)
```

Comment:
C'est très _potahto/potato_ mais en lisant "missing ..." ça me faire penser qu'on va remplir tout les trous alors qu'en fait on n'en crée une que si le trou est encore ouvert, j'sais pas si c'est clair ce que je veux dire :sweat_smile:.
```suggestion
            "found %d missed employee records notifications", len(employee_record_with_missing_notification)
```